docs/vectorizer-quick-start-voyage.md

```diff
@@ -0,0 +1,130 @@
+# Vectorizer quick start
+
+This page shows you how to create a vectorizer in a self-hosted Postgres instance. We also show how simple it is to do semantic search on the automatically embedded data!  
+We recommend to use voyage ai as an embedding provider, to follow this tutorial you need to have a voyage ai account and an api key. You can get one [here](https://voyageai.com/).
+
+## Setup a local development environment
+
+We use a docker-compose file to set up a development environment, it includes a:
```

Comment:
I changed the wording a bit, because the timescale image actually doesn't contain the worker and it truly is about the compose file in the quickstart. Hope that's fine with you :D